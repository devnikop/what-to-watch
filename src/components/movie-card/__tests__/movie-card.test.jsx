import React from "react";
import renderer from "react-test-renderer";

import MovieCard from "../movie-card.jsx";

const mock = {
  movie: {
    id: 1,
    src: `source`,
    title: `movie title`,
  },
};

it(`snapshot`, () => {
  const { movie } = mock;

  const tree = renderer
    .create(
      <MovieCard
        movie={movie}
        onClick={jest.fn()}
        onMouseEnter={jest.fn()}
        onMouseLeave={jest.fn()}
      />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
