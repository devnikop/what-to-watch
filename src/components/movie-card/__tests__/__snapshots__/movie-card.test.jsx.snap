// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<article
  className="small-movie-card catalog__movies-card"
  onMouseEnter={[Function]}
  onMouseLeave={[MockFunction]}
>
  <button
    className="small-movie-card__play-btn"
    onClick={[Function]}
    type="button"
  >
    Play
  </button>
  <div
    className="small-movie-card__image"
  >
    <img
      alt="movie title"
      height="175"
      src="source"
      width="280"
    />
  </div>
  <h3
    className="small-movie-card__title"
  >
    <a
      className="small-movie-card__link"
      href="movie-page.html"
    >
      movie title
    </a>
  </h3>
</article>
`;
